packages/react-data-grid/src/RowGroup.tsx

```diff
@@ -52,9 +52,9 @@ export default forwardRef<HTMLDivElement, Props<any>>(function RowGroup(props, r
       <Renderer {...props} ref={ref} onRowExpandClick={onRowExpandClick} onRowExpandToggle={onRowExpandToggle} />
     </div>
   );
-});
+}) as <R>(props: Props<R>) => JSX.Element;
```

Comment:
We do pass the ref but I am not too sure if it is needed or not. We need to check if `renderBaseRow` is needed anymore or not. IIRC `renderBaseRow` was added because there was no easy way to forward the ref earlier